dremio_client/model/data.py

```diff
@@ -527,6 +527,11 @@ def __delitem__(self, key):
     def remove(self):
         return delete_catalog(self._token, self._base_url, self.meta.id, self.meta.tag, self._ssl_verify)
 
+    def get_item(self, cid=None, path=None):
+        result = self._catalog_item(cid, path)
+        _, obj = create(result, self._token, self._base_url, self._flight_endpoint, ssl_verify=self._ssl_verify)
```

Comment:
hey @insatomcat I think this should live on `DremioClient` rather than on the `Catalog` object. On the `Catalog` object it would be available anywhere in the hierarchy which is a bit misleading

Reply:
done, tell me what you think